public/components/CampaignNotes/CampaignNotes.js

```diff
@@ -0,0 +1,61 @@
+import React, {Component, PropTypes} from 'react';
+import CampaignNotesList from '../CampaignNotesList/CampaignNotesList';
+import CampaignNotesAdd from './CampaignNotesAdd';
+
+class CampaignNotes extends Component {
+
+  componentWillMount() {
+    this.props.campaignNoteActions.getCampaignNotes(this.props.campaign.id);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    if (nextProps.campaign.id !== this.props.campaign.id) {
+      this.props.campaignNoteActions.getCampaignNotes(nextProps.campaign.id);
+    }
+
+  }
+
+  saveNote = () => {
+      this.props.campaignNotesAddActions.createNote(this.props.id, { content: this.state.message });
+      window.removeEventListener('click', this.disableAdding, true);
```

Comment:
This saveNote function is actually not used anywhere, I forgot to remove when i was moving stuff around. I've now deleted and the actual removal and adding of the event listener are both happening in the campaignNotesAdd component